code/src/components/Artist.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+const generateDelimiter = (index, length) => {
+  if (index === length - 2) {
+    return ' & ';
+  } else if (index < length - 2) {
+    return ', ';
+  }
+};
+
+export const Artist = (props) => {
+  console.log(props)
```

Comment:
console.log is very useful when working with the code but can be neat to remove it when finished.